roles/openshift_facts/molecule.yml

```diff
@@ -0,0 +1,20 @@
+---
+ansible:
+  playbook: tests/playbook.yml
+  group_vars:
+    test_group:
+    - deployment_type: origin
+dependency:
+  name: galaxy
+driver:
+  name: docker
+docker:
+  containers:
+  - name: test_host
+    image: centos/systemd
```

Comment:
I wasn't able to get the tests that managed service manipulation working without using the systemd image.